guides/source/form_helpers.md

```diff
@@ -644,19 +644,19 @@ Output:
 Uploading Files
 ---------------
 
-A common task is uploading some sort of file, whether it's a picture of a person or a CSV file containing data to process. File upload fields can be rendered with the [`file_field`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-file_field) helper. The most important thing to remember with file uploads is that the rendered form's `enctype` attribute **must** be set to "multipart/form-data". If you use `form_with` with `:model`, this is done automatically:
+A common task is uploading some sort of file, whether it's a picture of a person or a CSV file containing data to process. File upload fields can be rendered with the [`file_field`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-file_field) helper.
 
 ```erb
 <%= form_with model: @person do |form| %>
   <%= form.file_field :picture %>
 <% end %>
 ```
 
-If you use `form_with` without `:model`, you must set it yourself:
+The most important thing to remember with file uploads is that the rendered form's `enctype` attribute **must** be set to "multipart/form-data". This is done automatically if you use a `file_field` inside a `form_with`. You can also set the attribute manually:
 
 ```erb
 <%= form_with url: "/uploads", multipart: true do |form| %>
-  <%= form.file_field :picture %>
+  <%= file_field_tag :picture %>
```

Comment:
I think using the form builder is preferred in this guide. See 612eb35a5a98cb1424678304781ab1ae4ea7f7dc
```suggestion
  <%= form.file_field :picture %>
```

Reply:
> Ah ok, this is to show how it works without invoking the builder 👍

Yep that's right.